app/views/dashboards/_dashboard_article_row.html.erb

```diff
@@ -58,11 +58,11 @@
           <% end %>
         </span>
 
-        <% if false %>
-          <span class="flex items-center p-1 ml-1" title="Data">
+        <% if article.user_subscriptions.any? %>
```

Comment:
This is going to issue a db call every time we call it which is twice, here and below on line 98. We should cache the value and then apply it in each place. OR if we wanted to take our db calls from 3 to 1, we could just do `article.user_subscriptions.size` and then for the booleans check if size is greater than 0

Reply:
Darn, I thought `.any?` was smart enough to check it in memory since we eagerly loaded them already 🤔 . I'll cache it 😢 .

Reply:
@mstruve Updated to use `counter_culture` 🎉  Thanks for catching this!

Reply:
for the rationale behind any and the other methods https://www.speedshop.co/2019/01/10/three-activerecord-mistakes.html

unfortunately the whole deal is quite complicated :D